plugins/verification-plugin/src/main/kotlin/DetektPlugin.kt

```diff
@@ -13,7 +13,14 @@ class DetektPlugin : Plugin<Project> {
             target.extensions.configure<DetektExtension> {
                 buildUponDefaultConfig = true
                 baseline = target.file("detekt-baseline.xml")
-                config.from(target.rootProject.file("detekt.yml"))
+
+                val localDetektConfig = target.file("detekt.yml")
+                val rootDetektConfig = target.rootProject.file("detekt.yml")
```

Comment:
Does specifying both files always instead of checking existence work?

Reply:
looks like that works. good idea!

Reply:
scratch that... I didn't run it correctly locally when testing. reverted back to how it was.